packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,35 +146,36 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     if (isDisabled !== undefined) {
       disabled = isDisabled;
     }
-    const describedBy = id + '-option';
+    const describedby = id + '-option';
+    const labelledby = id + '-label';
 
     return (
       <div className={css('ms-Dropdown-container')}>
         {label && (
-          <Label className={css('ms-Dropdown-label')} id={id + '-label'} htmlFor={id} required={required}>
+          <Label className={css('ms-Dropdown-label')} id={labelledby} htmlFor={id} required={required}>
             {label}
           </Label>
         )}
         <KeytipData keytipProps={keytipProps} disabled={disabled}>
           {(keytipAttributes: any): JSX.Element => (
-            <div
+            <button
               {...keytipAttributes}
               data-is-focusable={!disabled}
               ref={this._dropDown}
               id={id}
               tabIndex={disabled ? -1 : 0}
               aria-expanded={isOpen ? 'true' : 'false'}
-              role="listbox"
-              aria-live={disabled || isOpen ? 'off' : 'assertive'}
               aria-label={ariaLabel}
-              aria-describedby={mergeAriaAttributeValues(describedBy, keytipAttributes['aria-describedby'])}
+              aria-labelledby={`${mergeAriaAttributeValues(labelledby, keytipAttributes['aria-labeledby'])}`}
+              aria-describedby={`${mergeAriaAttributeValues(describedby, keytipAttributes['aria-describedby'])}`}
```

Comment:
What effect does adding `${` have?

Reply:
Ah, looks like intermediate code that I didn't end up fully reverting